statsmodels/stats/rates.py

```diff
@@ -0,0 +1,200 @@
+'''Test for ratio of Poisson intensities in two independent samples
+
+Author: Josef Perktold
+License: BSD-3
+
+destination statsmodels
+
+'''
+
+
+from __future__ import division
+import numpy as np
+from scipy import stats
+
+
+# copied from statsmodels.stats.weightstats
+def _zstat_generic2(value, std_diff, alternative):
```

Comment:
remove, I only added this to have a standalone script that didn't depend on statsmodels
(for stackoverflow)